### PR TITLE
ASC-224 add space request

### PR DESCRIPTION
**Pull request by mfrancisc**
Add SpaceRequest CRD

Jira Ticket: [ASC-244](https://issues.redhat.com/browse/ASC-244)

Related PRs:

- api: https://github.com/codeready-toolchain/api/pull/333
- sandbox-sre: https://github.com/codeready-toolchain/sandbox-sre/pull/874
- toolchain-cicd: https://github.com/codeready-toolchain/toolchain-cicd/pull/78

### CODE REVIEW

File: make/run-cicd-script.mk [hunk 1]
```diff
@@ -1,5 +1,4 @@
-
-OWNER_AND_BRANCH_LOCATION=codeready-toolchain/toolchain-cicd/master
+OWNER_AND_BRANCH_LOCATION=mfrancisc/toolchain-cicd/feature/ASC-224_SpaceRequest
```
Comment (xcoulon) [inline make/run-cicd-script.mk:1]: reminder

File: go.mod [hunk 2]
```diff
@@ -100,4 +100,6 @@ require (
 	sigs.k8s.io/yaml v1.3.0 // indirect
 )
 
+replace github.com/codeready-toolchain/api => github.com/mfrancisc/api v0.0.0-20221219150138-6643d58aa259
```
Comment (xcoulon) [inline go.mod:103]: reminder

**Review by xcoulon — APPROVED** [review]

**Review by alexeykazakov — APPROVED** [review]

**Comment by mfrancisc** [comment]
something strange happened with the linter 🤔 

apparently it's not understanding anymore this import syntax:

```
import (
. "github.com/codeready-toolchain/member-operator/test"
)
```

**Review by MatousJobanek — APPROVED** [review]

File: make/run-cicd-script.mk (hunk 1 above)
Comment (mfrancisc) [reply make/run-cicd-script.mk]: thanks! fixed : [90600b3b220](https://github.com/codeready-toolchain/member-operator/pull/400/commits/90600b3b220f9773e595092ac1fe6ad752d63b28)

File: go.mod (hunk 2 above)
Comment (mfrancisc) [reply go.mod]: thanks! fixed : [90600b3b220](https://github.com/codeready-toolchain/member-operator/pull/400/commits/90600b3b220f9773e595092ac1fe6ad752d63b28)

**Pull request merged**